Tests/TimeTests/TimeSpec.swift

```diff
@@ -34,7 +34,7 @@ func testTime() {
         }
         $0.it("has a hashValue") {
             let interval = Interval<OneSecond>(10)
-            try expect(interval.hashValue) == interval.timeInterval.hashValue
+            try expect(interval.hashValue) != 0
```

Comment:
@foscomputerservices can you explain the reason behind this?

Reply:
Sure, no problem.

The issue is that now it is required to implement `public func hash(into hasher: inout Hasher) {}` instead of `public var hashValue: Int {}`.  Since the provided `Hasher` is now generating the hash, we can no longer assume what the value might be due to `hasher.combine()`.

Sorry, I thought I documented that, but I see that I did not.  Thanks for your awesome library!

Reply:
I see what you're saying. That's fine by me!